hack/versions-check.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+REPOS="prometheus prometheus-alertmanager node_exporter prometheus-operator kube-state-metrics thanos"
```

Comment:
k8s-prometheus-adapter
```suggestion
REPOS="prometheus prometheus-alertmanager node_exporter prometheus-operator kube-state-metrics thanos k8s-prometheus-adapter
"
```